### PR TITLE
Assignment 4 mandatory

### PR DESCRIPTION
**Pull request by joselucomino**

### CODE REVIEW

**Comment by ocorcho** [comment]
@dgarijo Please check it when you have time. The student has also made some changes to the notebooks, which you may want to accept if you wish, but has also provided the .py files.

File: Assignment4/joselucomino-a180436/Task07.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+# 
+
+# In[5]:
+
+
+# RDFLib
+for s,p,o in g.triples((None, RDF.type, ns.Person)):
+    print(s)
+for s1,p1,o1 in g.triples((None, RDFS.subClassOf, ns.Person)):
+    print(s1)
```
Comment (dgarijo) [inline Assignment4/joselucomino-a180436/Task07.py:59]: These are not individuals of Person. These are subclasses of person (line 59)

File: Assignment4/joselucomino-a180436/Task07.py [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+# SPARQL
+query1 = prepareQuery('''
+    SELECT ?subject ?predicate ?object WHERE {
+    {
+        ?subject rdf:type ns:Person.
+        ?subject ?predicate ?object
+    } UNION {
+        ?subject rdf:type ns:Person.
+        ?subject ?predicate ?object.
+        ?subject rdfs:subClassOf ns:Person
```
Comment (dgarijo) [inline Assignment4/joselucomino-a180436/Task07.py:103]: This is wrong. You usually cannot be subclass of and instance of the same class. You are looking for a query like:
?subject rdf:type/rdfs:subclassOf* ns:Person

**Comment by dgarijo** [comment]
@joselucomino please do not include changes to the notebooks. Also, please fix the issues with your tasks.

**Pull request closed without merging**